src/OAuth/OAuth1/Service/Flickr.php

```diff
@@ -100,34 +100,65 @@ public function request($path, $method = 'GET', $body = null, array $extraHeader
 
     public function requestRest($path, $method = 'GET', $body = null, array $extraHeaders = array())
     {
-        return $this->request($path, $method, $body, $extraHeaders);
+        return $this->requestAPI($path, $method, $body, $extraHeaders);
```

Comment:
`requestRest` does not set a `format` - if it were to take on the last `format` could it be potentially wrong?